ui/desktop/tests/acceptance/authentication-test.js

```diff
@@ -235,4 +249,108 @@ module('Acceptance | authentication', function (hooks) {
     assert.dom('[data-test-no-auth-methods]').includesText('No Auth Methods');
     assert.dom('.change-origin').exists();
   });
+
+  test('signing out with running sessions renders signout modal', async function (assert) {
+    this.ipcStub.withArgs('hasRunningSessions').returns(true);
+
+    await visit(urls.authenticate.methods.global);
+
+    await fillIn('[name="identification"]', 'test');
+    await fillIn('[name="password"]', 'test');
+    await click('[type="submit"]');
```

Comment:
Instead of signing in manually can we use `authenticateSession` function instead or is there a particular reason for this?

Reply:
Totally. No, no particular reason. I noticed this is what we are doing in other tests, so I implemented the same convention. Will change it.